.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -216,7 +226,7 @@ private object GutenbergUploadSourceMapsToSentry: BuildType() {
 					cd gutenberg
 
 					# Upload the .js and .js.map files to Sentry (`wpcom-test-01` release)
-					sentry-cli --auth-token %SENTRY_AUTH_TOKEN% releases --org a8c --project wpcom-gutenberg-wp-admin files wpcom-test-01 upload-sourcemaps . --url-prefix "~/wp-content/plugins/gutenberg-core/%GUTENBERG_VERSION%/"
+					sentry-cli --auth-token %SENTRY_AUTH_TOKEN% releases --org a8c --project wpcom-gutenberg-wp-admin files %SENTRY_RELEASE_NAME% upload-sourcemaps . --url-prefix "~/wp-content/plugins/gutenberg-core/%GUTENBERG_VERSION%/"
```

Comment:
Minor style thing -- to improve readability, could we do something like this? I found this command difficult to read.

```suggestion
					sentry-cli releases files %SENTRY_RELEASE_NAME% upload-sourcemaps . \
							--auth-token %SENTRY_AUTH_TOKEN% \
							--org a8c \
							--project wpcom-gutenberg-wp-admin \
							--url-prefix "~/wp-content/plugins/gutenberg-core/%GUTENBERG_VERSION%/"
```

(Assuming that the command still parses it correctly!)

Reply:
It doesn't look more readable here, but I see your point! :D Let me try committing this and seeing if it looks alright.

Reply:
Oh, looks much better from the regular diff viewer :)

Reply:
I also tested the command locally after the re-formatting, and it works well 👍🏻